cmd/thor/flags.go

```diff
@@ -59,6 +59,11 @@ var (
 		Name:  "api-allow-custom-tracer",
 		Usage: "allow custom JS tracer to be used tracer API",
 	}
+	apiLogsLimitFlag = cli.IntFlag{
```

Comment:
should we add it too docs, or it is hidden ?

Reply:
Will add to the docs, thanks!